mono/utils/Makefile.am

```diff
@@ -239,8 +241,8 @@ endif
 
 libmonoutils_la_SOURCES = $(monoutils_sources) $(arch_sources)
 
-libmonoutils_il2cpp_la_SOURCES = $(libmonoutils_la_SOURCES)
-libmonoutils_il2cpp_la_CFLAGS = -DIL2CPP_ON_MONO -DDISABLE_JIT -DDISABLE_REMOTING -fexceptions
+libmonoutils_unity_il2cpp_la_SOURCES = $(libmonoutils_la_SOURCES) $(unity_files)
+libmonoutils_unity_il2cpp_la_CFLAGS = -DIL2CPP_ON_MONO -DDISABLE_JIT -DDISABLE_REMOTING -fexceptions -DPLATFORM_UNITY -I../../../../il2cpp/build/libil2cpp/os/c-api
```

Comment:
shouldn't the CFLAGS be setup in the configure.ac file like GLIB_CFLAGS are 

GLIB_CFLAGS='-I$(top_srcdir)/eglib/src -I$(top_builddir)/eglib/src'
GLIB_LIBS='-L$(top_builddir)/eglib/src -leglib -lm'
  
AC_SUBST(GLIB_CFLAGS)
AC_SUBST(GLIB_LIBS)